src/Init/Data/Range/Polymorphic/PRange.lean

```diff
@@ -125,6 +125,27 @@ macro_rules
   | `($a<...<$b) => ``(PRange.mk (shape := RangeShape.mk BoundShape.open BoundShape.open) $a $b)
   | `($a<...$b) => ``(PRange.mk (shape := RangeShape.mk BoundShape.open BoundShape.open) $a $b)
 
+recommended_spelling "Rcc" for "a...=b" in [PRange, «term_...=_»]
```

Comment:
Would this make more sense as
```suggestion
recommended_spelling "Rcc" for "a...=b" in [PRange.mk, «term_...=_»]
```
throughout?

Reply:
I would guess that `recommended_spelling` only works for things that already have a docstring